tests/test_utils_kernel.py

```diff
@@ -18,11 +18,11 @@
 
 
 @pytest.mark.parametrize("ker", rker_fct)
-def test_kernel(ker, rndstate, get_covmats):
+def test_kernel_x_x(ker, rndstate, get_covmats):
     """Test Kernel build"""
     n_matrices, n_channels = 12, 3
```

Comment:
```suggestion
    n_matrices, n_dim = 12, 3
```
Do we keep n_channels here?

Reply:
Yes, you are right.
But I suggest to clean tests later (after issue on HPD compliance).